src/Host/Broker/Impl/Resources.resx

```diff
@@ -255,4 +255,7 @@
   <data name="Error_ExitRHost" xml:space="preserve">
     <value>RHost Exited with Win32 error : {0}</value>
   </data>
+  <data name="Text_ServiceName" xml:space="preserve">
+    <value>RHostBrokerService</value>
```

Comment:
The name here is the Service Name, e.g, seclogon. User friendly service name is Service Display Name. If you are suggesting that we use user friendly name for the event log, I can make that change. If we want to rename the display name for Broker Service, then i can do that. Should it be 'R Service' ?